core/arch/emit_utils_shared.c

```diff
@@ -2370,14 +2430,71 @@ append_call_dispatch(dcontext_t *dcontext, instrlist_t *ilist, bool absolute)
  *
  *  # append_save_simd_reg
```

Comment:
Isn't this in the wrong place?  It's now after the stack swap and xflags safe.

Reply:
I had only updated the restore comment. Fixed.